src/redux/sagas/motions/claimAllMotionRewards.ts

```diff
@@ -67,6 +74,17 @@ function* claimAllMotionRewards({
       throw new Error('A motion with claims needs to be provided');
     }
 
+    const colonyClient = yield colonyManager.getClient(
+      ClientType.ColonyClient,
+      colonyAddress,
+    );
+
+    const votingReputationClient: AnyVotingReputationClient = yield call(
+      [colonyManager, colonyManager.getClient],
+      ClientType.VotingReputationClient,
+      colonyAddress,
+    );
```

Comment:
This only seems to be used to get the voting reputation address, should we get it from the DB and pass it to the saga instead?